src/scripts/main.js

```diff
@@ -1,3 +1,324 @@
 'use strict';
 
-// write code here
+// TABLE_VARIABLES
+const table = document.querySelector('table');
+const tHead = table.tHead;
+const tBody = table.tBodies[0];
+
+// #region SORTING
+// SORTING FUNCTIONS
+const valueConversionFunctions = {
+  age: parseInt,
+  salary: window.convertSalartyToNumber,
```

Comment:
It appears that you have a typo in the function name `convertSalartyToNumber` on line 12. Please correct this to `convertSalaryToNumber`.